test/e2e/validation_test.go

```diff
@@ -464,6 +465,64 @@ func testExpectedServicesRunning(t *testing.T) {
 	}
 }
 
+// testServicesConfigMap tests 2 aspects of expected functionality for the services ConfigMap
+// 1. It exists on operator startup and 2. It is recreated when deleted
+func testServicesConfigMap(t *testing.T) {
+	tc, err := NewTestContext()
+	require.NoError(t, err)
+
+	err = tc.validateWindowsServicesConfigMap()
+	require.NoError(t, err, "error validating windows-instances ConfigMap")
+
+	err = tc.deleteAndWaitForServicesConfigMap()
+	require.NoError(t, err, "error ensuring windows-services ConfigMap is re-created when deleted")
+}
+
+// validateWindowsServicesConfigMap ensures the windows-services ConfigMap exists in the cluster and has valid contents
+func (tc *testContext) validateWindowsServicesConfigMap() error {
+	windowsServices, err := tc.client.K8s.CoreV1().ConfigMaps(tc.namespace).Get(context.TODO(),
+		wsvalidator.ServicesConfigMap, meta.GetOptions{})
+	if err != nil {
+		return errors.Wrapf(err, "error retrieving ConfigMap: %s", wsvalidator.ServicesConfigMap)
+	}
+
+	if _, _, err := wsvalidator.ParseAndValidate(windowsServices.Data); err != nil {
```

Comment:
What is the advantage in calling this function given this will be called by the operator code also? The real check should be if the operator is not throwing an error.